### PR TITLE
[new release] producer (0.2.0)

### PR DESCRIPTION
**Pull request by Sixstring982**
Accumulate results using monadic dependency graphs

- Project page: <a href="https://github.com/sixstring982/producer">https://github.com/sixstring982/producer</a>
- Documentation: <a href="https://sixstring982.github.io/producer">https://sixstring982.github.io/producer</a>

##### CHANGES:

* Pin dependency versions in `producer.opam` and `dune-project`
* Add support for monads with two type parameters (e.g. `Result`)

### CODE REVIEW

**Comment by Sixstring982** [comment]
Looks like there are some CI failures here, let me take a look.

File: packages/producer/producer.0.2.0/opam [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+authors: ["Trent Small"]
+license: "BSD-3-Clause"
+tags: ["dependency injection" "graph" "node" "producer"]
+homepage: "https://github.com/sixstring982/producer"
+doc: "https://sixstring982.github.io/producer"
+bug-reports: "https://github.com/sixstring982/producer/issues"
+depends: [
+  "ocaml" {>= "4.08"}
+  "dune" {>= "3.7" & >= "3.7"}
+  "alcotest" {"1.7.0" >= with-test}
```
Comment (avsm) [inline packages/producer/producer.0.2.0/opam:13]:
```suggestion
  "alcotest" {>="1.7.0" & with-test}
```

Should address the CI failures.

**Comment by avsm** [comment]
Thanks! You may want to announce this on <https://discuss.ocaml.org>, where we have a Community category and an `announce` tag for this purpose.

**Comment by Sixstring982** [comment]
Excellent -- thanks for the fix, @avsm!

Also -- noted, I'll take a look at https://discuss.ocaml.org/ !

**Pull request merged**